cartographer/mapping_2d/local_trajectory_builder.cc

```diff
@@ -37,11 +37,11 @@ LocalTrajectoryBuilder::LocalTrajectoryBuilder(
 LocalTrajectoryBuilder::~LocalTrajectoryBuilder() {}
 
 sensor::RangeData LocalTrajectoryBuilder::TransformAndFilterRangeData(
-    const transform::Rigid3f& gravity_alignment,
+    const transform::Rigid3f& transform,
```

Comment:
Choose better names? This is not an arbitrary transform but needed so that min and max z make sense. `gravity_alignment` was also used in other places as well, so that helped the reader to understand that it is this transform. Please also see https://google-cartographer.readthedocs.io/en/latest/terminology.html for the frame it related to.